api/v1/lib/builders.go

```diff
@@ -2,49 +2,65 @@ package mesos
 
 type (
```

Comment:
not sure that this file belongs in the mesos package, maybe in a `builders` package (stuffed into `extras`)?